lib/sudo-pam/src/converse.rs

```diff
@@ -35,17 +35,20 @@ impl PamMessageStyle {
     }
 }
 
+use crate::Password;
+type RawBytes = Vec<u8>;
```

Comment:
This typedef feels unnecessary for me, I think `Vec<u8>` is understood well enough by general Rust programmers as raw byte data.